components/dashboard/src/workspaces/StartWorkspaceModal.tsx

```diff
@@ -58,6 +61,17 @@ export function StartWorkspaceModal(props: StartWorkspaceModalProps) {
         return true;
     }, [repo, selectedIde, selectedWsClass, useLatestIde, errorIde, errorWsClass]);
 
+    if (isNewCreateWorkspacePage) {
```

Comment:
Is the intent of putting this inside `StartWorkspaceModal` to make it easier to make all the existing places that start a workspace to run through here? Do you think we'd eventually be able to remove this `StartWorkspaceModal` and replace the locations opening this modal with just a link to the new page?

Reply:
Yes, it's here as kind of a catch all until we have fully rolled out the new CreateWorkspaceFlow.